webhook.go

```diff
@@ -28,6 +30,43 @@ type DeliveryConfig struct {
 	AlternativeURLs []string `json:"alt_urls,omitempty"`
 }
 
+// WebhookConfig is a Webhook substructure with data related to event delivery.
+type WebhookConfig struct {
+	// Accept is content type of outgoing events. The following content types are supported, otherwise
+	// a 406 response code is returned: application/octet-stream, application/jsonl, application/msgpack.
+	Accept string `json:"accept"`
+
+	// Secret is the string value.
+	// (Optional, set to "" to disable behavior).
+	Secret string `json:"secret,omitempty"`
+
+	// SecretHash is the hash algorithm to be used. Only sha256 HMAC and sha512 HMAC are supported.
+	// (Optional).
+	// The Default value is the sha512 HMAC.
```

Comment:
support all sha (greater than 256) in go std
use the highest sha as the default